### DISCUSSION TITLE
USPS Mail Pieces and Packages pulling same value?

### DISCUSSION THREAD

**Discussion by drac0linux** [moralmunky/Home-Assistant-Mail-And-Packages#955] [General]
Hi, first off, thanks for the integration. It's helpful. 

According to my Informed Delivery email from the USPS, I have three pieces of mail that will be arriving today. I also have two packages from Amazon which the Informed Delivery email states will arrive today. Amazon packages are reflected correctly under Amazon Packages, however under the USPS sensors, both the mail and packages values are the same.

![image](https://github.com/user-attachments/assets/25bd9ce1-c45a-457e-9c43-4330141ca362)

Is this by design? Should the Packages and Pieces counts be the same?

Thanks!

**Comment by firstof9** [comment]
> Should the Packages and Pieces counts be the same?

Only if you have the same count of mail and packages coming. According to your 'delivering' sensor, you have 3 packages out for delivery, so you have a packages count of 3.

**Reply by drac0linux** [reply]
Well, that's interesting. I'll have to say that you are right, because after all my looking at this earlier this morning, the number of packages did update from two to three. Thanks!
